### PR TITLE
[wip] fixes to content, bugs in figure display

### PR DESCRIPTION
**Pull request by ValHayot**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -543,7 +540,7 @@ \subsection{Datasets} % 1/4 of a page (half a col)
 partitioned into 125 chunks.
 
 Processing large images is only considered to be part of the Big Data problem 
-in neuroscience. The other problem is processing large MRI datasets, 
+in neuroscience. The other being the processing large MRI datasets,
```
Comment (glatard) [inline paper.tex]: is a word missing here?

File: paper.tex [hunk 2]
```diff
@@ -734,9 +731,9 @@ \subsection{Experiments}
 chunks was fixed to 125. Similarly to the previous experiment, the 
 total sequential compute time was fixed (10 iterations, 1.76 seconds 
 per task), however, due to varying size in total data processed (D), 
-the D/C ratio varied. Once again, we ensured that the D/C ratio fell in 
-multiple different regions of the inequality. The D/C ratio ranged from 
-349.1 MB/s for BigBrain and 174.6MB/s for half Bigbrain, to 0.06MB/s for the MRI 
+the data-write rate varied. Once again, we ensured that the data-write rate fell in
```
Comment (glatard) [inline paper.tex]: I'd keep `D/C` to make it explicit. Maybe "the D/C data-write rate"

File: paper.tex [hunk 3]
```diff
@@ -956,13 +958,13 @@ \subsection{Effect of In-Memory Computing}
 computing on the pre-processing of functional MRI, another typical use 
 case in neuroimaging. Assuming an average processing time of 20 minutes 
 per subject, which is a ballpark value commonly observed with the 
-popular \href{https://www.fil.ion.ucl.ac.uk/spm/}{SPM} or \href{https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/}{FSL packages},
+popular \href{https://www.fil.ion.ucl.ac.uk/spm/}{SPM} or \href{https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/}{FSL} packages,
```
Comment (glatard) [inline paper.tex]: remove 's' in 'packages'.

File: paper.tex [hunk 4]
```diff
@@ -44,53 +44,50 @@
...
-computing, adopted by MapReduce's successor Apache 
-Spark~\cite{zaharia2016apache}, stores data in memory whenever 
-possible, reducing costly disk I/Os between pipeline steps. 
-Lazy evaluation, also used in Spark, enables further Big Data performance 
-optimizations, by evaluating results only when necessary. Frameworks 
-such as
-MapReduce and Spark have become mainstream tools for data analytics, 
-although many others, such as 
-Dask~\cite{rocklin2015dask}, are emerging. 
+In Big Data applications, there is a significant cost associated with data movement.
```
Comment (glatard) [inline paper.tex]: This first sentence reads as a tautology, I find it less powerful than the previous one. You could merge it with the second one.

File: paper.tex [hunk 5]
```diff
@@ -44,53 +44,50 @@
...
+Big Data processing engines have addressed these issues by diminishing the amount 
+of data movement that occurs during the execution of an application. The
```
Comment (glatard) [inline paper.tex]: I'd say 'data overhead' instead of 'data movement' because 'overhead' covers both 'I/O' and actual movement between nodes.

File: paper.tex (hunk 1 above)
Comment (ValHayot) [reply paper.tex]: neuroscience is the last noun, but 'problem' is the only noun that can have an 'other'. if it's not clear, i can add 'problem' back

File: paper.tex [hunk 6]
```diff
@@ -44,53 +44,50 @@
...
     Neuroimaging datasets are rapidly growing in size as a result of 
     advancements in image acquisition methods, open-science and data sharing. 
-    However, it remains that current neuroimaging data processing engines do 
-    not employ strategies adopted by Big Data processing engines to improve 
-    processing time. Here, we 
-    evaluate three Big Data processing stategies (in-memory computing, 
-    data-locality and lazy-evaluation) on typical neuroimaging use 
-    cases, represented by the BigBrain dataset. We contrast the various 
-    processing strategies by using Apache Spark and Nipype as our 
+    However, current neuroimaging data engines have not adopted any strategies
```
Comment (glatard) [inline paper.tex]: "have not adopted any" is a bit harsh

File: paper.tex [hunk 7]
```diff
@@ -44,53 +44,50 @@
...
+were found to increase the likelihood of cache hits, further improving
+    processing time. Such important 
 speed-up values are likely to be observed on typical image processing 
-operations performed on images of size larger than 75GB. A ballpark 
-speculation from our model showed that in-memory computing alone will
-not speed-up current 
-functional MRI analyses unless coupled with data 
-locality and processing around 280 subjects concurrently. 
-    In addition, we observe that emulating in-memory computing 
-using in-memory file systems (tmpfs) does not reach the performance of
```
Comment (glatard) [inline paper.tex:71]: why did you remove that? I find it important.

File: paper.tex [hunk 8]
```diff
@@ -44,53 +44,50 @@
...
+when combined with data locality, this factor reaches 5.3. Lazy evaluation
+strategies
+were found to increase the likelihood of cache hits, further improving
+    processing time. Such important 
 speed-up values are likely to be observed on typical image processing 
-operations performed on images of size larger than 75GB. A ballpark 
-speculation from our model showed that in-memory computing alone will
-not speed-up current 
-functional MRI analyses unless coupled with data 
-locality and processing around 280 subjects concurrently.
```
Comment (glatard) [inline paper.tex:69]: I find it important to mention fmri too.

File: paper.tex (hunk 1 above)
Comment (glatard) [reply paper.tex]: ok then
